lib/csv_to_popolo.rb

```diff
@@ -339,30 +353,72 @@ def given?(key)
       @r.key?(key) && !@r[key].nil? && !@r[key].empty?
     end
 
+    def split_cell(key)
+      separator = MODEL[key][:multivalue_separator]
+      if separator
+        @r[key].split(separator)
+      else
+        [@r[key]]
+      end
+    end
+
+    def set_cell_from_values(key, values)
```

Comment:
I was actually uneasy about that existing code, since it means there'd be different results depending on the order you call methods on `Person`, so I'll figure out another way.